auth-web/src/components/auth/CreateAccount/AccountCreateBasic.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <v-form ref="createAccountInfoForm">
+    <div class="view-container">
+      <h1 class="mb-5">Account Settings</h1>
+      <v-alert type="error" v-show="errorMessage">{{ errorMessage }}</v-alert>
+      <h4 class="mb-5">Enter an Account Name</h4>
+      <v-text-field
+        filled
+        label="Account Name"
+        v-model.trim="teamName"
+        :rules="teamNameRules"
+        persistent-hint
+        :disabled="saving"
+        :hint="
+          teamType === 'BASIC'
+            ? 'Example: Your Business Name'
+            : 'Example: Your Management Company or Law Firm Name'
+        "
+      />
+      <v-row>
+        <v-col cols="12" class="d-inline-flex">
+          <v-btn large color="grey lighten-3" class="mx-1" @click="goBack">
+            <v-icon left class="mr-1">mdi-arrow-left</v-icon>
+            Back
+          </v-btn>
+          <v-spacer></v-spacer>
+          <v-btn
+            large
+            color="primary"
+            class="mr-2"
+            :loading="saving"
+            :disabled="!isFormValid() || saving"
+            @click="save"
+            data-test="save-button"
+          >
+            Save and Continue
+          </v-btn>
+          <v-btn
+            large
+            depressed
+            color="default"
+            :disable="saving"
+            @click="cancel"
+            data-test="cancel-button"
+          >
+            Cancel
+          </v-btn>
+        </v-col>
+      </v-row>
+    </div>
+  </v-form>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { CreateRequestBody, Member, Organization } from '@/models/Organization'
+import { mapActions, mapState } from 'vuex'
+import BaseAddress from '@/components/auth/BaseAddress.vue'
+import BcolLogin from '@/components/auth/BcolLogin.vue'
+import OrgModule from '@/store/modules/org'
+import { getModule } from 'vuex-module-decorators'
+
+@Component({
+  components: {
+    BcolLogin,
+    BaseAddress
+  },
+  computed: {
+    ...mapState('org', ['currentOrganization']),
+    ...mapState('user', ['userProfile', 'currentUser'])
+  },
+  methods: {
+    ...mapActions('org', ['createOrg', 'syncMembership', 'syncOrganization'])
+  }
+})
+export default class AccountCreateBasic extends Vue {
+  private orgStore = getModule(OrgModule, this.$store)
+  private teamName: string = ''
+  private teamType: string = 'BASIC'
+  private errorMessage: string = ''
+  private saving = false
+  private readonly createOrg!: (
+    requestBody: CreateRequestBody
+  ) => Promise<Organization>
+  private readonly syncMembership!: (orgId: number) => Promise<Member>
+  private readonly syncOrganization!: (orgId: number) => Promise<Organization>
+  private readonly currentOrganization!: Organization
+  @Prop() stepForward!: () => void
+  @Prop() stepBack!: () => void
```

Comment:
I have a PR pending that once merged, you will be able to just extend a mixin `Steppable` that will remove the need to manually specific these.